doc/source/ray-core/ray-dashboard.rst

```diff
@@ -106,6 +106,21 @@ The log viewer provides various search functionality to help find the log messag
 Advanced Usage
 --------------
 
+Viewing built-in dashboard API metrics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The API server that serves the dashboard also emits basic prometheus metrics for API call count and API call duration.
```

Comment:
@alanwguo actually I checked the doc and unfortunately there's no mentioning of this "API server" concept. Can we somehow either link to more details or describe this in a different way? Otherwise users will be confused about what this API server refers to.